cob_hardware_config/common/upload_robot.launch

```diff
@@ -1,13 +1,34 @@
 <?xml version="1.0"?>
 <launch>
 
-    <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
-    <arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+	<arg name="robot_description" default="robot_description"/>
```

Comment:
what's the use case for naming this different than robot_description? If there is no use case, we should remove that argument